.github/Feature_Request.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
This looks great Max! Just a couple of considerations below.

It looks like this file should be added to the `.github/ISSUE_TEMPLATE/` directory:

> To be included in the community profile checklist, issue templates must be located in the .github/ISSUE_TEMPLATE folder
>
> https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates

Let's also follow the Wagtail project example and, for example, use all upper-case letters for the file name with a lowercase .md file extension:

https://github.com/wagtail/wagtail/tree/master/.github/ISSUE_TEMPLATE